ComposableArchitecture/Project.swift

```diff
@@ -36,6 +36,19 @@ let p = Project(
       infoPlist: infop,
       sources: ["App/Sources/**"],
       resources: ["App/Resources/**"]
-    )
+    ),
+    Target(
```

Comment:
I am guessing these changes are required because we are using `Tuist`.

Reply:
yep! This is declarative development, where instead of committing the project after creating the target in Xcode, we commit the declaration of the target and use Tuist to generate it.